Complete aesthetic overhaul

orengeblaster opened the pull request:
## Currently done:
- [x] New stall model

## Work in progress:
- [x] Props inside the stall
- [x] Customer sprites
- [x] Stall surrounding
- [x] Placeholder assets removal

orengeblaster committed:
updated stall

gongpha committed:
improve model storing

gongpha committed:
^^ brighter world ^^

(experimental)

orengeblaster committed:
remove mainmanu_background,tscn

orengeblaster committed:
add asset submodule

gongpha committed:
adjust item spawning behavior

gongpha committed:
change a clock behavior

gongpha commented:
Note : you must merge all opening pull requests before merging this pull request.

gongpha committed:
move the blueberry robot to snap on the wall

gongpha committed:
new customer animations

gongpha committed:
new the end of the day screen

gongpha committed:
implement pausemenu

gongpha committed:
fix when pressing a confirm key and escape key

gongpha committed:
add simple audios

from my old project 💀

gongpha committed:
Timer starts when the dialogue ends

gongpha committed:
try setuping some music

gongpha committed:
smooth trimesh shape ?

gongpha committed:
fix timer glitches

gongpha committed:
fix when a player is trying to sell the same item

gongpha committed:
shorten a holding duration

from 1.33s to 0.80s

gongpha committed:
move the player a bit

MAYBE NOT THE RIGHT SOLUTION. PLEASE CORRECT ME

gongpha committed:
import assets

made by @jolluxx and @I-GUN-I
imported by me

Co-Authored-By: jollux <52147209+jolluxx@users.noreply.github.com>
Co-Authored-By: Gun... <117667567+I-GUN-I@users.noreply.github.com>

gongpha committed:
move the mainmenu camera a bit

the vision just gets blocked by newly imported assets lmao

gongpha committed:
Merge branch 'aesthetic-overhaul' into gameplay-refactoring-1

gongpha committed:
FIX WORLDSPAWN CONFLICTS

gongpha committed:
improve transition ( ? )

gongpha committed:
add building

"gm_construct"

gongpha committed:
add lamps

orengeblaster committed:
rename items

gongpha committed:
Merge branch 'aesthetic-overhaul' into gameplay-refactoring-1

gongpha committed:
Merge pull request #3 from gongpha/gameplay-refactoring-1

Gameplay Refactoring #1

gongpha committed:
remove a player vision observer

gongpha committed:
better item randomizer

and move a stool little a bit

gongpha committed:
remove the finance system

gongpha committed:
add ranking

Co-Authored-By: Gun... <117667567+I-GUN-I@users.noreply.github.com>

gongpha committed:
Merge pull request #5 from gongpha/gameplay-refactoring-2

Gameplay Refactoring #2

gongpha committed:
Add various customer sprites

gongpha committed:
Fix the variable name typo

gongpha committed:
fix lamp transparent part

gongpha committed:
Add the monitor behavior

gongpha committed:
fix the customer sprites

gongpha committed:
improve the font

gongpha committed:
fix when blocking a picking action

gongpha committed:
adjust auto exposure & glow values

gongpha committed:
change names' position in the credit screen

gongpha committed:
add american english as placeholders

gongpha committed:
reformat dialogues

gongpha committed:
add indignantly when you keep repeating

gongpha committed:
move the monitor to the game node

gongpha committed:
remove 3d placeholders

gongpha committed:
add loading screen

gongpha committed:
remove the main theme

and placed in individual nodes. prevent preloading before the loading screen

gongpha committed:
adjust the counter area

gongpha committed:
add our font

designed by @jolluxx

Co-Authored-By: jollux <52147209+jolluxx@users.noreply.github.com>

gongpha committed:
improve ui

orengeblaster committed:
updated font (unpatched)

orengeblaster committed:
add newline to game.py

orengeblaster committed:
add item name in in-game language

gongpha committed:
repatch the font

gongpha committed:
adjust space for main menu rows

orengeblaster committed:
remove unused module

gongpha committed:
add cheat mode

for debugging purposes.
access by holding the F1 key for 3 seconds

gongpha committed:
fix endday logics

gongpha committed:
cheat mode meme

nkarGn committed:
change menu to blue

nkarGn committed:
add game intro

gongpha committed:
redirect game to the intro scene

gongpha committed:
fix overlapping

nkarGn committed:
Revert "change menu to blue"

This reverts commit 77292613f279c144aa5e59dcfcfbb998fdfdb770.

orengeblaster committed:
remove fridge door

nkarGn committed:
Improve intro scene a bit

gongpha committed:
spawn milk in the fridge instead of on the shelf

nkarGn committed:
add sounds

gongpha committed:
apply conlang in item names only

nkarGn committed:
add mainmenu fx

gongpha committed:
add city ambience

gongpha committed:
add impact detector (gd)

gongpha committed:
play the sound fx when confirming

gongpha committed:
playing sound when changing the volume

nkarGn committed:
add footstep

nkarGn committed:
add npc voice

gongpha committed:
add cheats guidelines

I-GUN-I committed:
darken

I-GUN-I committed:
make the monitor easier to read

gongpha committed:
add customer beep effect

nkarGn committed:
shorten npc vo

gongpha committed:
ugly removal

gongpha committed:
Rename CHEATS.MD to CHEATS.md

orengeblaster committed:
add proper dialogue

orengeblaster committed:
Merge branch 'final-stretch' of https://github.com/gongpha/pscp-python-project-2022 into final-stretch

gongpha committed:
fix conflicts and reformatting dialogues

damn

gongpha committed:
increase times

nkarGn committed:
add mainmenu bgm

gongpha committed:
move BGMs

gongpha committed:
add endday music

gongpha committed:
add loading label

gongpha committed:
try adding emoticons

it's too hard to understand their emotion via the dialogue but we can know what are they feeling about our actions

gongpha committed:
Merge branch 'final-stretch' into worldspawn-atmosphere

gongpha committed:
Merge pull request #4 from gongpha/worldspawn-atmosphere

Audio Compatibility

gongpha committed:
reset all items when a new day begins

gongpha committed:
blocking impact sounds when spawning items

gongpha committed:
move endday screen nodes

gongpha committed:
Merge pull request #6 from gongpha/final-stretch

The Final Stretch

orengeblaster committed:
add more translator dialogue

gongpha committed:
normalize mainmenu

gongpha committed:
translator notifications

gongpha committed:
highlight item texts again

gongpha committed:
fix button texts

gongpha committed:
fix pausemenu

gongpha committed:
Update dialogue.py

gongpha committed:
fix milk

gongpha committed:
‼️ WARNING ‼️

TIME IS RUNNING OUT

gongpha committed:
show the rank table on every days

gongpha committed:
Update game.py

gongpha commented:
Game footage recorded at 3894c4ff37ccb03342eed3ed3c2ec9040c22e9b4

gongpha committed:
Merge branch 'master' into aesthetic-overhaul

gongpha committed:
FINAL

- conflicts
- the stool rotates along with player
- highlight item names at the begin
- fix grammo
- restart the day when got ranked "E"
- format python
- PSCP Project Grouppe

gongpha reviewed (approved):
lgtm, but I really love the cheating boom effect part ROFLMAO
